issue #251

dexink opened the pull request:
All changes agreed upon during OMRSE meeting, 5 March 2025, including all four new terms proposed by @CDowland (issue #251)

dexink committed:
issue #251

All changes agreed upon during OMRSE meeting, 5 March 2025, including all four new terms proposed by @CDowland (issue #251 )

hoganwr reviewed (changes requested):
The identifiers for the new classes you created are not OBO compliant. You cannot use UUIDs. We can work through the required Protege settings to create proper IDs.

You cannot change the definitions of IAO classes. We need to check that all IAO is being imported and that there is no longer any copy and paste into omrse-edit.owl.

hoganwr commented:
Apologies, I take back the bit about IAO class definitions.

We were going to do the obsoletion of the old stuff in this PR, or a separate one?

dexink committed:
Revisions to PR #269

## Primary Changes
### The following changes were made with proper Protege settings applied to PR #269 branch omrse-edit.owl file:
_*Changes under this section are restricted to the addition of the requested and approved new terms, ref. issue #251; see screencaptures, labeled A-G, of each item below_

1. Recreation of class 'social category data item' (OMRSE:00000292) [A]
2. Recreation of class 'racial category data item' as child class of 'social category data item' (OMRSE:00000295) [B]
3. Recreation of class 'gender category data item' as child class of 'social category data item' (OMRSE:00000294) [C]
4. Recreation of class 'ethnic category data item' as child class of 'social category data item' (OMRSE:00000293) [D]

### For adequate axiom support of new classes (above), the following were required:

1. Recreation of class 'gender category' (OMRSE:00000291) [E]
2. Recreation of class 'gender categorization scheme' (OMRSE:00000290) [F]
3. Creation of class 'gender identity' (OMRSE:00000296) [G]

## Screencaptures
**A**
![Screenshot 2025-03-25 at 9 19 01 AM](https://github.com/user-attachments/assets/ba16bf30-54bd-4b0b-8fb7-6212e3c9d116)

**B**
![Screenshot 2025-03-25 at 9 19 40 AM](https://github.com/user-attachments/assets/299de655-0042-4193-9834-f5d06ebca4c4)

**C**
![Screenshot 2025-03-25 at 9 20 09 AM](https://github.com/user-attachments/assets/15bbb1ce-c8eb-4667-8203-e3793d8e5746)

**D**
![Screenshot 2025-03-25 at 9 20 46 AM](https://github.com/user-attachments/assets/f0ad0935-82ef-4c71-9e6d-a1d8a146c5c8)

**E**
![Screenshot 2025-03-25 at 9 21 14 AM](https://github.com/user-attachments/assets/27fd7cb3-67c0-4102-b611-9965b78cfc4a)

**F**
![Screenshot 2025-03-25 at 9 21 50 AM](https://github.com/user-attachments/assets/9ba2e6f8-9e65-4f9c-b410-9d32698b23e3)

**G**
![Screenshot 2025-03-25 at 9 22 18 AM](https://github.com/user-attachments/assets/ebaeec75-9cad-4f6b-b495-9af62ada9b2a)

dexink commented:
## Primary Changes
### The following changes were made with proper Protege settings applied to PR #269 branch omrse-edit.owl file:
_*Changes under this section are restricted to the addition of the requested and approved new terms, ref. issue #251; see screencaptures, labeled A-G, of each item below_

1. Recreation of class 'social category data item' (OMRSE:00000292) [A]
2. Recreation of class 'racial category data item' as child class of 'social category data item' (OMRSE:00000295) [B]
3. Recreation of class 'gender category data item' as child class of 'social category data item' (OMRSE:00000294) [C]
4. Recreation of class 'ethnic category data item' as child class of 'social category data item' (OMRSE:00000293) [D]

### For adequate axiom support of new classes (above), the following were required:

1. Recreation of class 'gender category' (OMRSE:00000291) [E]
2. Recreation of class 'gender categorization scheme' (OMRSE:00000290) [F]
3. Creation of class 'gender identity' (OMRSE:00000296) [G]


## Screencaptures
**A**
![Screenshot 2025-03-25 at 9 19 01 AM](https://github.com/user-attachments/assets/ba16bf30-54bd-4b0b-8fb7-6212e3c9d116)

**B**
![Screenshot 2025-03-25 at 9 19 40 AM](https://github.com/user-attachments/assets/299de655-0042-4193-9834-f5d06ebca4c4)

**C**
![Screenshot 2025-03-25 at 9 20 09 AM](https://github.com/user-attachments/assets/15bbb1ce-c8eb-4667-8203-e3793d8e5746)

**D**
![Screenshot 2025-03-25 at 9 20 46 AM](https://github.com/user-attachments/assets/f0ad0935-82ef-4c71-9e6d-a1d8a146c5c8)

**E**
![Screenshot 2025-03-25 at 9 21 14 AM](https://github.com/user-attachments/assets/27fd7cb3-67c0-4102-b611-9965b78cfc4a)

**F**
![Screenshot 2025-03-25 at 9 21 50 AM](https://github.com/user-attachments/assets/9ba2e6f8-9e65-4f9c-b410-9d32698b23e3)

**G**
![Screenshot 2025-03-25 at 9 22 18 AM](https://github.com/user-attachments/assets/ebaeec75-9cad-4f6b-b495-9af62ada9b2a)

hoganwr committed:
fixed dc annotations. Supposed to use dcterms from now on, and dcterms:contributor is enforced by ODK to be an IRI, so you have to convert the ORCID to an IRI/URL 

hoganwr reviewed (approved):
This looks great! I updated the contributor annotations. We should use dcterms from now on, and dcterms:contributor requires an IRI so you need to use the URL form of ORCID (note that URL subClassOf IRI).
